### DISCUSSION TITLE
Upgrading PicFrame

### DISCUSSION THREAD

**Discussion by ghost** [helgeerbe/picframe#141] [Q&A]
Hi. Would you mind telling me (or pointing me to) the recommended steps to upgrading Picframe to the lastest version?  I've not been able to find anything on this project page, likely missed it.

Thanks
Richard

**Comment by sapnho** [comment]
Richard, That's how I do it: https://www.thedigitalpictureframe.com/pi3d-faq-image-viewer-picture-frame/#How_can_I_update_my_Pi3D_PictureFrame_version

**Reply by ghost** [reply]
Fantastic, thanks sapnho.  And for your quick reply!

**Reply by helgeerbe** [reply]
I point out in the [changelog](https://github.com/helgeerbe/picframe/wiki/Changelog) if there are changes to `configuration.yaml`. But  I alway recommend to make a backup of the config file.

**Comment by ghost** [comment]
Good to know, thanks.
